### PR TITLE
create nsswitch.conf file to resolve unknownhost exception issue

### PR DESCRIPTION
**Pull request by apratina**
In absence of /etc/nsswitch.conf file it uses /etc/hosts (file) or /etc/resolv.conf (dns) to resolve hostname. Using /etc/nsswitch.conf we can define the order in which host name resolution should be done. More details on the following [blog](https://community.hpe.com/t5/System-Administration/nsswitch-conf/m-p/3815887/highlight/true#M269190).

On looking further into this issue some folks have also reported similar issue in java+alpine combo and here is the suggested fix.
 - http://arthur-c.github.io/posts/alpine-linux-java-unknownhostexception/
 - http://stackoverflow.com/questions/32537977/java-net-unknownhostexception-on-docker?answertab=votes#tab-top
 - https://github.com/gliderlabs/docker-alpine/issues/11#issuecomment-91329401
 - https://github.com/gliderlabs/docker-alpine/issues/11#issuecomment-106233554
 
Basically we need to write /etc/nsswitch.conf file with following entry.
```
hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4
```

To understand the exact meaning of the above line you can read [here](https://ubuntuforums.org/showthread.php?t=971693&p=12589927#post12589927)
Some more details on nsswitch.conf file is available [here](http://man7.org/linux/man-pages/man5/nsswitch.conf.5.html).

### CODE REVIEW

**Comment by apratina** [comment]
@bryanlatten : Please take a look at this PR.

**Comment by bryanlatten** [comment]
👍

**Comment by bryanlatten** [comment]
@ko-be @bossjones?

**Comment by ko-be** [comment]
👍

**Comment by bossjones** [comment]
👍
